irony.el

```diff
@@ -99,6 +99,19 @@ automatically buffer-local wherever it is set."
   "C/C++ minor mode powered by libclang."
   :group 'c)
 
+(defcustom irony-allow-inaccessibles nil
```

Comment:
Hello,

I did not use this variable because I was unsure how to use it in the filtering function.

I presume you mean that not-available candidates can be left out for now since a derived's protected member is deemed not-accessible, not not-available?